### DISCUSSION TITLE
无法访问https://higress.io/helm-charts/，返回404

### DISCUSSION THREAD

**Discussion by Ironeie** [alibaba/higress#1499] [General]
helm add repo后helm pull, helm show value皆为空，访问https://higress.io/helm-charts/返回404

**Comment by CH3CHO** [comment]
https://higress.io/helm-charts/index.yaml

不过你访问它干啥呢？

如果上面链接不行的话，你换成 https://higress.cn/helm-charts/index.yaml 试试
